Sources/_SwiftSyntaxTestSupportFrameworkAgnostic/String+TrimmingTrailingWhitespace.swift

```diff
@@ -15,7 +15,14 @@ extension String {
   public func trimmingTrailingWhitespace() -> String {
     return
       self
-      .replacingOccurrences(of: "[ ]+\\n", with: "\n", options: .regularExpression)
-      .trimmingCharacters(in: [" "])
```

Comment:
Previous implementation would have also trimmed space from the start of the string, I'm assuming that was unintentional given the name of the method?

Reply:
Yes, I agree. I don’t know why we were also trimming spaces from the start and the end of the string.